.eslintrc

```diff
@@ -1,25 +1,29 @@
-{
-  "parser"  : "babel-eslint",
-  "extends" : [
-    "airbnb"
-  ],
-  "plugins": [
-    "babel",
-    "react"
-  ],
-  "env"     : {
-    "browser" : true
-  },
-  "globals" : {
-    "__DEV__"      : false,
-    "__PROD__"     : false,
-    "__DEBUG__"    : false,
-    "__COVERAGE__" : false,
-    "__BASENAME__" : false
-  },
-  "rules": {
-    "generator-star-spacing": 0,
-    "max-len": [2, 120, 2],
-    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }]
-  }
-}
```

Comment:
Note I changed this file from JSON to YAML. JSON is probably preferable, but with YAML we can structure the rules into groups using comments. Since JSON doesn't support comments I think this is an advantage but happy to revert to JSON if there's a better reason for it.